docs/en/00_Getting_Started/02_Composer.md

```diff
@@ -56,48 +64,57 @@ see [Using development versions](#using-development-versions).
 
 Composer isn't only used to download SilverStripe CMS, it can also be used to manage all SilverStripe modules.  Installing a module can be done with the following command:
 
-	composer require "silverstripe/forum:*"
+```
+composer require "silverstripe/forum:*"
+```
+
+This will install the forum module in the latest compatible version. You can also omit the `:*` if you simply want the latest compatible version of the module.
```

Comment:
>IMHO using "*" as a version is a bad practice.

While I agree completely, I think this example is trying to cater for people who don't know which major version they want to install.

Perhaps we should encourage them to update their constraint in `composer.json` and update again once they've found the latest stable version, or just omit the `*` to install (and track) the latest version by default?